src/com/hrabrov/introduction_to_java/basic_of_software_code_development/cycle/Task02.java

```diff
@@ -1,30 +1,27 @@
 package com.hrabrov.introduction_to_java.basic_of_software_code_development.cycle;
 
-import java.util.Arrays;
-import java.util.ArrayList;
-
 public class Task02 {
     /**
      * Found value function on the segment [a,b] with increment h:
-     * y = x, x > 2 || y = -x, x <= 2.
+     * y = x, x > 2 && y = -x, x <= 2.
      *
-     * @param a input initial value segment
+     * @param a input start value segment
      * @param b input finish value segment
      * @param h input increment
      * @return value function
      */
 
     public static String foundFunctionValue(double a, double b, double h) {
-        ArrayList<Double> arrayValueFunction = new ArrayList<>();
+        String valueFunction = "";
 
         for (double i = a; i <= b; i += h) {
             if (i == 0) {
-                arrayValueFunction.add(0.0);
+                valueFunction += "0 ";
             } else {
-                arrayValueFunction.add(i > 2 ? i : -1 * i);
+                valueFunction += i <= 2 ? Double.toString(-i) + " " : Double.toString(i) + " ";
```

Comment:
Why do you use `String` there?

Reply:
I will return 2 and more. how can i return this without use String and array?